src/components/forms/inputs/selectInput.tsx

```diff
@@ -29,6 +29,7 @@ interface SelectInputProps<T>
   placeholder?: string;
   showError?: boolean;
   theme?: string;
+  optionsEditable?: boolean;
```

Comment:
I will remove it since it is not needed anymore. Previously I updated the options according to the given project, So the options were dynamic, now they are static.